src/storage/src/hummock/sstable/builder.rs

```diff
@@ -534,6 +563,17 @@ impl<W: SstableWriter, F: FilterBuilder> SstableBuilder<W, F> {
             }
         };
 
+        let vnode_bitmap_mapping = {
+            let mut mapping = HashMap::default();
+
+            for (table_id, vnode_bitmap_builder) in self.vnode_bitmap_mapping {
+                let buf = vnode_bitmap_builder.finish().to_protobuf();
+                mapping.insert(table_id, buf);
+            }
+
+            mapping
+        };
```

Comment:
We should use `None`` for the mapping if enable_vnode_bitmap is false.

Reply:
ditto, can't make the map type optional, do we need to replace the map with a custom type?